taxi/tests/test_views.py

```diff
@@ -0,0 +1,218 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Car
+
+INDEX_URL = reverse("taxi:index")
+MANUFACTURER_URL = reverse("taxi:manufacturer-list")
+CAR_URL = reverse("taxi:car-list")
+DRIVER_URL = reverse("taxi:driver-list")
+CAR_DETAIL_URL = reverse("taxi:car-detail", args=[1])
+
+
+class PublicIndexTest(TestCase):
+    def test_loggin_required(self):

```

Comment:
There's a typo in the method name `test_loggin_required`. It should be `test_login_required` for clarity and correctness.